tests/test_session_filtering.py

```
@@ -111,32 +115,305 @@ def sample_traces(self):
+            "/api/v1/traces?has_media=true",
+            headers={"Authorization": "Bearer namastex888"}
+        )
+        
+        # Verify the response
+        assert response.status_code == 200
+        data = response.json()
+        assert len(data) == 1  # Only trace_004 has media
+        
+        # Verify all returned traces have media
+        for trace in data:
+            assert trace["has_media"] == True
```

Ruff (0.12.2)

Avoid equality comparisons to `True`; use `trace["has_media"]:` for truth checks

Replace with `trace["has_media"]`

(E712)